src/main/java/com/google/sps/data/TrickNode.java

```diff
@@ -0,0 +1,16 @@
+package com.google.sps.data;
+
+// a class that holds data pertaining to a trick
+public class TrickNode {
+  public String trick_name;
+  public long date;
+  public String link;
+  public String notes;
```

Comment:
Typically class attributes are made private and get/set methods are used for each attribute